contracts/lottery/Makefile.toml

```diff
@@ -0,0 +1,281 @@
+[env]
+friendbot = "https://friendbot-futurenet.stellar.org/"
+network = "futurenet"
+server = "https://horizon-futurenet.stellar.org/"
+
+[env.futurenet]
+friendbot = "https://friendbot-futurenet.stellar.org/"
+network = "futurenet"
```

Comment:
Probably we should start migrating to testnet, as Soroban is already available there :)

I think there is no date for removing futurenet. But will happen soon.

Reply:
I can add another environment.